scriptworker/task.py

```diff
@@ -311,6 +360,29 @@ def get_repo_scope(task, name):
         return repo_scopes[0]
 
 
+def extract_github_repo_owner_and_name(repo_url):
+    """Given an URL, return the repo name and who owns it.
+
+    Args:
+        repo_url (str): The URL to the GitHub repository
+
+    Raises:
+        ValueError: on repo_url that aren't from github
+
+    Returns:
+        str, str: the owner of the repository, the repository name
+
+    """
+    if not repo_url.startswith('https://github.com/'):
+        raise ValueError('{} is not a supported GitHub URL!'.format(repo_url))
+
+    if repo_url.endswith(".git"):
+        repo_url = repo_url[:-len(".git")]
+    elements = repo_url.split('/')
```

Comment:
There are enough weird ways to specify urls that it may be safest to use [`urlparse`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse) and then act on the path.

Reply:
Fixed!